src/app/sass-sections/movies.scss

```diff
@@ -0,0 +1,440 @@
+.movies {
```

Comment:
This is the file from the last project right? I think we don't use everything in here so maybe we could trim this down a bit. But I guess it's OK as it is also.

Reply:
Yes. I'll leave it for now since it works. Maybe we need some of that styling later on. Else i'll remove it.